capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -31,23 +80,22 @@ class ClubPage extends Component {
   }
 
   componentDidMount() {
-    this.fetchClubData();
-  }
-
-  componentDidUpdate(prevProps) {
-    if (this.props.match.params.id !== prevProps.match.params.id) {
-      this.fetchClubData();
-    }
+    this.fetchData();
   }
 
   render() {
+    const bookTile = this.state.book.authors && <BookSearchTile book={this.state.book} userBookLists={this.state.bookLists} />;
+    const owner = this.state.owner && <div> Club Owner: {this.state.owner.fullName}, {this.state.owner.email} </div>;
```

Comment:
There will be a conflict here if we merge our PRs. 

You have to pass the booklists props from App.jsx in order for BookSearchTile to work properly. I can resolve this conflict if you merge before me.

Reply:
Sure, I'll leave it as is and when you merge you can pass it down as a prop along with the function to perform the update.